ArduCopter/mode_auto.cpp

```diff
@@ -408,6 +415,81 @@ void ModeAuto::circle_start()
     }
 }
 
+Vector3f Mode::_LoiterTurns::get_circle_edge_neu() const
```

Comment:
I wonder if we should call this `_Circling` rather than `_LoiterTurns`.  So if someone wants to "loiter unlimited" we can use the same object without renaming.  The `_Takeoff` object isn't `_NAV_TAKEOFF` or whatever :-)

I thin kthe fact all the methods used to be prepended with `circling_` is a fair sign of that.

Reply:
Renamed it to `_Circling`. Thanks for your views :-)